source/reference/minio-mc/mc-idp-ldap-accesskey-disable.rst

```diff
@@ -0,0 +1,102 @@
+.. _minio-mc-idp-ldap-accesskey-disable:
+
+=================================
+``mc idp ldap accesskey disable``
+=================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+.. mc:: mc idp ldap accesskey disable
+
+
+Description
+-----------
+
+.. start-mc-idp-ldap-accesskey-disable-desc
+
+:mc:`mc idp ldap accesskey disable` disables the specified access key on the local server.
```

Comment:
local server -> MinIO deployment?

Reply:
```suggestion
:mc:`mc idp ldap accesskey disable` disables the specified access key on the MinIO deployment.

This command has no affect on access keys associated to non-AD/LDAP users.
```

Reply:
Also maybe link access key to `minio-id-access-keys` ?

Reply:
Not sure if line 24 is necessary - technically AD/LDAP should disable MinIO IAM, but the root user can still have access keys.